### DISCUSSION TITLE
Getting a cookie error in Lighthouse. Any ideas?

### DISCUSSION THREAD

**Discussion by jasondainter** [plausible/analytics#3010] [Anything Else]
Hi Folks. 

Am using Plausible and keep getting this issue logged to dev tools in chrome. 

Issue type
--
Cookie
/js/script.js(plausible.io)

Its on a next.js site Ive built, any ideas how to fix or what this means?

<img width="718" alt="Screenshot 2023-06-07 at 18 15 42" src="https://github.com/plausible/analytics/assets/1300757/194bd121-53ec-46c9-8426-517cb24b17ba">

**Comment by metmarkosaric** [comment]
thanks for reaching out @jasondainter! please do contact our support as it's difficult to troubleshoot something like this without getting more info such as your website domain. see https://plausible.io/contact

**Reply by jasondainter** [reply]
Sure thanks I've moved over to email.
